src/v/cluster/cloud_metadata/cluster_manifest.cc

```diff
@@ -136,12 +161,14 @@ cluster_metadata_manifest::get_manifest_path() const {
 std::ostream& operator<<(std::ostream& os, const cluster_metadata_manifest& m) {
     os << fmt::format(
       "{{upload_time_since_epoch: {}, cluster_uuid: {}, metadata_id: {}, "
-      "controller_snapshot_offset: {}, controller_snapshot_path: '{}'}}",
+      "controller_snapshot_offset: {}, controller_snapshot_path: '{}', "
+      "offsets_snapshots_by_partition: '{}'}}",
       m.upload_time_since_epoch,
       m.cluster_uuid,
       m.metadata_id,
       m.controller_snapshot_offset,
-      m.controller_snapshot_path);
+      m.controller_snapshot_path,
+      m.offsets_snapshots_by_partition);
```

Comment:
praise: never realized that the ranges formatter could handle vector<vector<string>> but now it's obvious